paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -481,8 +482,6 @@ internal class DefaultFlowController @Inject internal constructor(
             return
         }
 
-        val initializationMode = requireNotNull(initializationMode)
```

Comment:
I didn't feel good about adding another one of these to present. So I did a quick remodeling to make it available without this!